examples/dgmulti_2d/elixir_euler_bilinear.jl

```diff
@@ -2,7 +2,7 @@
 using Trixi, OrdinaryDiffEq
 
 dg = DGMulti(polydeg = 3, element_type = Quad(), approximation_type = SBP(),
-             surface_integral = SurfaceIntegralWeakForm(flux_hll),
+             surface_integral = SurfaceIntegralWeakForm(FluxHLL(min_max_speed_naive)),
```

Comment:
We should discuss whether we want to keep the naive version in most elixirs (simpler) or update it to the commonly used `flux_hll` version (requires more work to update tests).

Reply:
I add this to the agenda for the next Trixi meeting.